indexer/feature_meta.cpp

```diff
@@ -71,7 +71,7 @@ bool Metadata::TypeFromString(string_view k, Metadata::EType & outType)
     outType = Metadata::FMD_CONTACT_FACEBOOK;
   else if (k == "instagram" || k == "contact:instagram")
     outType = Metadata::FMD_CONTACT_INSTAGRAM;
-  else if (k == "twitter" || k == "contact:twitter")
+  else if (k == "twitter" || k == "contact:twitter" || k == "contact:x")
```

Comment:
Please remove it from here and modify `data/replaced_tags.txt` instead.

Reply:
Are you sure? There are no other contact tags in replaced_tags, and iirc the types are assigned in this function

Reply:
Until the OSM community decides to introduce contact:x, we will replace it with contact:twitter. If community decides to migrate and use contact:x as an official tag, then we will react accordingly.

Reply:
for sure, i'm just not sure if it will actually work in replaced_tags :) i'll just remove it